lib/spack/docs/getting_started.rst

```diff
@@ -183,7 +183,8 @@ You can verify that the new settings are effective with:
    When bootstrapping from sources, Spack requires a full install of Python
    including header files (e.g. ``python3-dev`` on Debian), and a compiler
    with support for C++14 (GCC on Linux, Apple Clang on macOS) and static C++
-   standard libraries on Linux.
+   standard libraries on Linux. Ensure to load Python modules *before* sourcing
+   the Spack ``setup-env`` script.
```

Comment:
What do you mean by "load Python modules"?

Reply:
The compute cluster on which these instructions were tested has a Python interpreter but without development files. There are several Python modules though and all of them contain the development files. When `spack-env.sh` is sourced, the _current_ Python interpreter path is stored in the `SPACK_PYTHON` environment variable. Thus, loading a Python module after sourcing `spack-env.sh` has thus no effect on Spack. This was not obvious to me because `which python3` was showing me the expected output.